api/tests/DataPersister/CampCollaborationsDataPersisterTest.php

```diff
@@ -1,198 +0,0 @@
-<?php
```

Comment:
@BacLuc Habe gerade gemerkt, dass du den Test für den CampCollaborationDataPersister komplett gelöscht hast. Bewusst oder unbewusst?
Ich habe gerade die ContentNodes refactored (#2195) und dabei die Tests drin gelassen.

Reply:
Das war bewusst, weil die Logik (if statements) nicht mehr wirklich da drin sind, sondern in der Abstraktion.